### DISCUSSION TITLE
Is it possible to specify the build environment?

### DISCUSSION THREAD

**Discussion by pkaeding** [goreleaser/goreleaser#2204] [Q&A] [answered]
From reading the docs, I'm not sure if it is possible to specify an alternate build environment. Is the actual build done in a docker container? (eg, `docker run golang:1.15 go build ...`)

Is it possible to specify an alternate container, such as [goboring/golang](https://hub.docker.com/r/goboring/golang/), or some other custom image that has golang, along with other tools needed for things like transpiling frontend code, etc)?

**Comment by caarlos0** [comment, marked as answer]
the build environment is whatever env you are running goreleaser into... docker is not used to build the binaries per se (unless, of course, you run goreleaser from within a docker container).
